### PR TITLE
fix: snapshot api break change

### PR DESCRIPTION
**Pull request by zhouhanseng**

### CODE REVIEW

File: packages/maskbook/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "@openzeppelin/contracts": "^3.2.0",
     "@popperjs/core": "*",
     "@servie/events": "^3.0.0",
-    "@snapshot-labs/snapshot.js": "^0.1.8",
+    "@snapshot-labs/snapshot.js": "github:snapshot-labs/snapshot.js#master",
```
Comment (Jack-Works) [inline packages/maskbook/package.json:35]: Does this package need a build step? If so, please build it and pack the result as a tgz file and install from tgz.

File: packages/maskbook/src/plugins/Snapshot/Worker/apis/index.ts [hunk 2]
```diff
@@ -35,13 +66,16 @@ export async function fetch3BoxProfiles(addresses: string[]): Promise<Profile3Bo
...
+export async function getScores(message: ProposalMessage, voters: string[], blockNumber: number, _network: string) {
     const spaceKey = message.space
     const strategies = message.payload.metadata.strategies
-    const network = ChainId.Mainnet.toString()
+    // Sometimes `message.payload.metadata.network` is absent, this is maybe a snapshot api issue.
+    const network = message.payload.metadata.network ?? _network
     const provider = ss.utils.getProvider(network)
     const snapshot = Number(message.payload.snapshot)
     const blockTag = snapshot > blockNumber ? 'latest' : snapshot
+    console.log({ provider, network, strategies, spaceKey, snapshot, blockTag })
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/Snapshot/Worker/apis/index.ts:77]: Remove this line?

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/maskbook/package.json (hunk 1 above)
Comment (zhouhanseng) [reply packages/maskbook/package.json:35]: No, [snapshot website](https://github.com/snapshot-labs/snapshot/blob/develop/package.json#L30) use the same 'github:snapshot-labs/snapshot.js#master'

Comment (Jack-Works) [reply packages/maskbook/package.json:35]: how do u think @guanbinrui

Comment (guanbinrui) [reply packages/maskbook/package.json:35]: Why not use the newest ver? [v0.1.9](https://registry.npmjs.org/@snapshot-labs/snapshot.js/-/snapshot.js-0.1.9.tgz).

Comment (zhouhanseng) [reply packages/maskbook/package.json:35]: It's easier to debug that I can read their code when keep the same with [snapshot website](https://github.com/snapshot-labs/snapshot/blob/develop/package.json#L30)

Comment (guanbinrui) [reply packages/maskbook/package.json:35]: Better not to use pkg from gh directly. It's easy to break.

Comment (Jack-Works) [reply packages/maskbook/package.json:35]: Actually it's version locked in the lock file by commit hash

Comment (guanbinrui) [reply packages/maskbook/package.json:35]: @Jack-Works I see. There is no problem with me.

**Review by Jack-Works — APPROVED** [review]

**Review by septs — APPROVED** [review]

**Pull request merged**